concepts/nil/about.md

```diff
@@ -17,7 +17,7 @@ In turn this means that the compiler will automatically check for null reference
 
 To be able to handle `nil` values, are there certain approaches that can be taken.
 `nil` is a [falsey][truthy-falsey] value as well are `false`.
-This means that in if statements and other places where a falsey or truthy value is expected it will be treated as false.
+This means that in if statements and other places where a falsey or truthy value is expected so will it be treated the same as `false`.
```

Comment:
```suggestion
This means that in if statements and other places where a falsey or truthy value is expected it will be treated the same as `false`.
```